src/components/menu/menu.tsx

```diff
@@ -80,34 +134,110 @@ export class Menu {
      * Is emitted when a menu item is selected.
      */
     @Event()
-    private select: EventEmitter<MenuItem | MenuItem[]>;
+    public select: EventEmitter<MenuItem | MenuItem[]>;
+
+    @Event()
+    public parentSelect: EventEmitter<MenuItem>;
```

Comment:
I know I'm reviewing stuff that's not been put up for review yet, sorry about that, but I wanted to understand this PR before trying to review #2482. Anyway, all these new props and events really need to get some descriptions, so one can understand what they are for 😊

Reply:
Hopefully fixed :)